decredPoS/decredPoS_script.md

```diff
@@ -1,43 +1,40 @@
 
 A ticketholder's ballot will contain one **block vote** and zero to many **consensus votes** to be recorded *on-chain*.
 
 Block votes allow stakeholders to keep proof-of-work miners in check. The 5 block votes contained in each new block decide whether to accept or reject the previous one. If stakeholders decide to reject the block, its pending transactions are returned to the mempool and the miner is stripped of their block reward.
 
-Consensus votes allow stakeholders to prevent unwanted hard-forks through a two stage process where stakeholders vote for or against changes to Decred's consensus rules.
+Consensus votes allow stakeholders to change Decred's consensus rules through a two stage process where stakeholders vote for or against the proposed change.
 
-Before the consensus voting process can begin, a majority of nodes must upgrade to the newly released software version containing the proposed rule change. The update threshold is met when 95% of the 1000 most recent blocks and 75% of the proof of stake votes within *one* **stake version interval**, 2016 blocks (or about 1 week), have the latest version. 
+Before the consensus voting process can begin, nodes must update to new software that has a latent copy of the proposed new rules. If the proposed rule change is approved, these will be automatically activated. The update threshold is met when 95% of the 1000 most recent blocks were mined by updated nodes and 75% of the proof of stake votes within *one* **stake version interval**, 2016 blocks (or about 1 week), have the latest version. 
 
 After the update threshold is met voting will begin on the *first block* of the next **rule change interval**. 
 
-Each rule change interval spans 8,064 blocks (or about 4 weeks) during which consensus votes for any active rule change proposals are collected.  A revote occurs If 90% of votes collected during the entire interval are abstaining.  Else, if 75% of non-abstaining votes signal yes, then the proposed rule changes will be activated at the end of the *next* rule change interval.
+Each rule change interval spans 8,064 blocks (or about 4 weeks) during which consensus votes for any active rule change proposals are collected.  For the outcome of the vote to be valid, at least 10% of the tickets must vote yes or no, as opposed to abstaining. If 75% of non-abstaining votes signal yes, then the proposed rule changes are approved and will be activated at the end of the *next* rule change interval. 
+
+If more than 90% of tickets abstain, or the 75% Yes or No majority is not achieved, the proposal will be voted on again in the next rule change interval. 
```

Comment:
...or *if* the 75%...

maybe it's also confusing to talk about "proposal" here (since that's linked to pi). I would rather stick with "changes" or similar